dbms/src/IO/tests/CMakeLists.txt

```diff
@@ -82,3 +82,6 @@ target_link_libraries (zlib_ng_bug PRIVATE ${Poco_Foundation_LIBRARY})
 if(NOT USE_INTERNAL_POCO_LIBRARY)
     target_include_directories(zlib_ng_bug SYSTEM BEFORE PRIVATE ${Poco_INCLUDE_DIRS})
 endif()
+
+add_executable(bit_io bit_io.cpp)
+target_link_libraries (bit_io PRIVATE clickhouse_common_io)
```

Comment:
Now we prefer using gtest for unit tests. They will be built into single executable and run in CI.
(You may convert to gtest right now or in separate PR after merging if you want.)